.circleci/config.yml

```diff
@@ -299,6 +299,22 @@ jobs:
             yarn storybook:ui:chromatic
       - store_test_results:
           path: test-results
+  chromatic-storybook-blocks:
+    executor:
+      class: medium+
+      name: sb_node_16_browsers
+    steps:
+      - git-shallow-clone/checkout_advanced:
+          clone_options: '--depth 1 --verbose'
+      - attach_workspace:
+          at: .
+      - run:
+          name: Running Chromatic
+          command: |
+            cd code
+            yarn storybook:blocks:chromatic
+      - store_test_results:
+          path: test-results
```

Comment:
This is the exact same setup as the `chromatic-storybook-ui` above it. We could squash them together since they are pretty related, but we'd loose the concurrency.

Although I doubt these jobs will be the bottleneck so maybe ditching the concurrency in favor of a simpler CI is worth it?

Reply:
Yeah, I think doing it in serial in one job probably makes sense.